packages/opentelemetry-instrumentation-anthropic/tests/test_bedrock_with_raw_response.py

```
@@ -53,10 +53,10 @@ async def test_async_anthropic_bedrock_with_raw_response(
 
     anthropic_span = spans[0]
     assert (
-        anthropic_span.attributes[f"{SpanAttributes.LLM_PROMPTS}.0.content"]
+        anthropic_span.attributes[f"{GenAIAttributes.GEN_AI_PROMPT}.0.content"]
```

Ruff (0.12.2)

Undefined name `GenAIAttributes`

(F821)